Devices/lilygo-tdisplay-s3/Source/Init.cpp

```
@@ -28,7 +27,7 @@ static bool powerOn() {
         return false;
     }
```

Cppcheck (2.19.0)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)